ethereum/core/src/main/java/org/hyperledger/besu/ethereum/bonsai/BonsaiWorldStateUpdater.java

```diff
@@ -639,6 +649,13 @@ private void rollStorageChange(
     }
   }
 
+  private boolean isSlotEquals(final UInt256 expectedValue, final UInt256 existingSlotValue) {
+    final UInt256 sanitizedExpectedValue = (expectedValue == null) ? UInt256.ZERO : expectedValue;
+    final UInt256 sanitizedExistingSlotValue =
+        (existingSlotValue == null) ? UInt256.ZERO : existingSlotValue;
```

Comment:
Optional.ofNullable is more readable than ternary's IMO :)

Reply:
oops I wanted to make the change but you were faster than me :sweat_smile: